transports/noise/src/lib.rs

```diff
@@ -213,8 +239,23 @@ pub enum Error {
     InvalidPayload(#[from] DecodeError),
     #[error(transparent)]
     SigningError(#[from] libp2p_identity::SigningError),
+    #[error("Unknown WebTransport certhashes: {}", certhashes_to_string(.0))]
```

Comment:
These are not the unknown WebTransport certhashes, but the ones expected but not received certhashes, right?

```suggestion
    #[error("Expected but not received WebTransport certhashes: {}", certhashes_to_string(.0))]
```

That would be more intuitive for me. I don't feel strongly about the concrete wording.

Reply:
With this wording, for me it feels that we need to provide all the expected certhashes, even the received ones, not just the subset.

Reply:
How should I proceed on this? These are my questions:

1. If I change the wording to `Expected but not received` should I print **all** the provided certhashes or just the unknown ones?
2. In case of multiple certhashes, should they be in the same other the user provided them in the multiaddr?

Reply:
> If I change the wording to `Expected but not received` should I print **all** the provided certhashes or just the unknown ones?

What do you think of the following?

```suggestion
    #[error("Expected WebTransport certhashes ({}) are not a subet of received ones ({})", certhashes_to_string(.0), certhashes_to_string(.0)]
    UnknownWebTransportCerthashes(HashSet<Multihash<64>>, HashSet<Multihash<64>>),
```